pruner/find.go

```diff
@@ -18,7 +18,7 @@ func (s *Service) findPruneableHeaders(
 	ctx context.Context,
 	lastPruned *header.ExtendedHeader,
 ) ([]*header.ExtendedHeader, error) {
-	pruneCutoff := time.Now().UTC().Add(time.Duration(-s.window))
+	pruneCutoff := time.Now().UTC().Add(-s.window.Duration())
```

Comment:
They added a `.Sub` that can be easier to process in your head vs Add on a negative number if you fancy that